great_expectations/datasource/fluent/sources.py

```diff
@@ -128,7 +128,10 @@ def register_datasource(cls, ds_type: Type[Datasource]) -> None:
             )
 
         # rollback type registrations if exception occurs
-        with cls.type_lookup.transaction() as ds_type_lookup, ds_type._type_lookup.transaction() as asset_type_lookup:  # fmt: skip # noqa: E501
```

Comment:
The paren syntax was added in python 3.9, so we were previously forced to prevent ruff from autoformatting it, since our netlify build includes sphinx parsing the AST.